### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by wintleh**
This PR creates the home page for this app. This PR also adds the basic templates for all other pages to use the header and footer.

### CODE REVIEW

File: src/definitions.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>EKGViewer - Definitions</title>
+    <link href="./css/reset.css" rel="stylesheet" />
+    <link href="./templates/css/page-template.css" rel="stylesheet" />
+    <link href="./css/definitions.css" rel="stylesheet" />
+</head>
```
Comment (wintleh) [inline src/definitions.php:9]: Each page uses the reset css, the page-template css, and its own css files. The reset css removes the styling added by browsers (makes everything consistent). The page-template css allows the header and footers to be styled correctly.

File: src/definitions.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+<body>
+    <?php include "./templates/page-header.php" ?>
+    <section id="content-wrapper"></section>
+    <?php include "./templates/page-footer.php" ?>
+</body>
```
Comment (wintleh) [inline src/definitions.php:15]: The body pulls the header and footer from the templates. The main page content goes in the "content-wrapper" section tag.

File: src/index.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    <title>Welcome to EKGViewer</title>
+    <link href="./css/reset.css" rel="stylesheet" />
+    <link href="./templates/css/page-template.css" rel="stylesheet" />
+    <link href="./css/index.css" rel="stylesheet" />
+</head>
+
+<body>
+    <?php include "./templates/page-header.php" ?>
+    <section id="content-wrapper">
+        <article id="welcome-wrapper">
```
Comment (wintleh) [inline src/index.php:14]: Filler data for now

**Comment by am2021** [comment]
Looks good, merging.

**Pull request merged**